src/components/Card/Card.js

```diff
@@ -1,11 +1,12 @@
 import React, { Component } from 'react'
+import propTypes from 'prop-types'
 
 export default class Card extends Component {
   constructor() {
     super()
     this.state = {
       flipped: false,
-      unflipped: false,
+      unflipped: true,
```

Comment:
This might be the only thing I could think of that would have change with the cards for the data issue we had. Don't know why that would change it really, but I'll pull this down and see.

Reply:
Nevermind, I think I figured it out. Something on my end I had to change to make the fonts work...